src/screenbot.js

```diff
@@ -24,20 +29,46 @@ var Screenbot = (function Screenbot() {
              channel_token;
     };
 
-    var _request = function(endpoint, callback){
+    var _request = function(endpoint, cb){
       var xhttp = new XMLHttpRequest();
       xhttp.onreadystatechange = function() {
         if (xhttp.readyState === 4 && xhttp.status === 200) {
           var response = JSON.parse(xhttp.responseText);
-          callback(response.ok, response);
+          if(!response.ok) return cb(response);
+          cb(null, response);
         } else if(xhttp.readyState === 4) {
-          callback(false, xhttp.responseText);
+          var response = JSON.parse(xhttp.responseText);
```

Comment:
It's best to have `try {} catch {}` around `JSON.parse`, especially when in a callback like this, because parse errors will not be caught sent to the callback otherwise. Same for the case above in line 36.